include/itkObjectnessMeasureImageFilter.h

```diff
@@ -23,9 +23,30 @@
 namespace itk
 {
 /** \class ObjectnessMeasureImageFilter
+ * \brief Enhance M-dimensional objects in N-dimensional images.
  *
- * This is a composite filter which combines a computation of the
- * hessian with computation of the objectness.
+ * This filter is a generalization of Frangi's vesselness measurement
+ * for detecting M-dimensional object in N-dimensional space. For
+ * example a vessel is a 1-D object in 3-D space. The filter
+ * can enhance blob-like structures (M=0), vessel-like structures
+ * (M=1), 2D plate-like structures (M=2), hyper-plate-like structures
+ * (M=3) in N-dimensional images, with M<N.
+ *
+ * This filter takes a scalar image as input and produces an real
+ * valued image as output which contains the objectness measure at
+ * each pixel. Internally, it computes a Hessian via discrete central
+ * differences. It is recommended to apply a Gaussian smoothing filter
+ * at a selected scale or sigma to capture the desired object feature.
+ *
```

Comment:
Please add equations 10,11,12 from Frangi's original paper here. The names alpha for R_a, beta for R_b and gamma for S are fine, but without the equations here users won't know what R_a, R_b, S are and will need to read the original paper (not available to everyone).

Reply:
I'm going to add the equation from the IJ paper.